presto-kafka/pom.xml

```diff
@@ -14,6 +14,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <dep.kafka.version>2.4.1</dep.kafka.version>
```

Comment:
this is used once, so  no need for a property

Reply:
done